dsc/examples/variables.dsc.yaml

```diff
@@ -0,0 +1,15 @@
+$schema: https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2024/04/config/document.json
+parameters:
+  myParameter:
+    type: string
+    # the use of `concat()` here is just an example of using an expression for a defaultValue
+    defaultValue: "[concat('world','!')]"
```

Comment:
There is a difference between parameters and variables:
parameters have 2 subfields: `type` and `defaultValue`,
but looks like variables have just implied `defaultValue` and type is implicitly derived.
Should this be consistent?

Reply:
I took this directly from ARM.  `variables` are intended to just be used to keep values that get reused multiple times within the config and typically constructed from params based on examples I saw.